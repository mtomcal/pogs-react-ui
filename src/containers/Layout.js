import React, {Component} from 'react';
import PropTypes from 'prop-types';
import Navbar from './Navbar';

class Layout extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    return (
      <div>
          <Navbar />
          <div className="container">
              <div className="row">
                  <div className="col-sm-10 col-sm-pull-1 col-sm-push-1">
                      {this.props.children}
                  </div>
              </div>
          </div>
      </div>
    );
  }
}

Layout.propTypes = {
    children: PropTypes.element
};

export default Layout;
