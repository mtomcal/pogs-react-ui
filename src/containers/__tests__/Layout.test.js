import React from 'react';
import Layout from '../Layout';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

it('renders with one element', () => {
  const component = renderer.create(
    <Layout>
      <h1>Stuff</h1>
    </Layout>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

