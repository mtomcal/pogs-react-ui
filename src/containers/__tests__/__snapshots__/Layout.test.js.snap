exports[`test renders with one element 1`] = `
<div>
  <div>
    <nav
      className="navbar navbar-default">
      <div
        className="container-fluid">
        <div
          className="row col-sm-12">
          <div
            className="navbar-header" />
        </div>
      </div>
    </nav>
  </div>
  <div
    className="container">
    <div
      className="row">
      <div
        className="col-sm-10 col-sm-pull-1 col-sm-push-1">
        <h1>
          Stuff
        </h1>
      </div>
    </div>
  </div>
</div>
`;
